packages/patternfly-4/react-table/src/components/Table/Table.api.test.js

```diff
@@ -41,6 +41,62 @@ describe('Selectable table', () => {
   });
 });
 
+describe('Action table', () => {
+  test('should call correct functions', () => {
```

Comment:
maybe this test is superfluous?

Reply:
do you lose coverage without it? It could be, but i don't see any harm if you are trying to ensure that these props are correctly digested.

Reply:
ok I will leave it in then